mne/time_frequency/tfr.py

```diff
@@ -1137,156 +1434,151 @@ def save(self, fname, overwrite=False):
         write_tfrs(fname, self, overwrite=overwrite)
 
 
-def _prepare_write_tfr(tfr, condition):
-    """Aux function"""
-    return (condition, dict(times=tfr.times, freqs=tfr.freqs,
-                            data=tfr.data, info=tfr.info,
-                            nave=tfr.nave, comment=tfr.comment,
-                            method=tfr.method))
-
+class EpochsTFR(_BaseTFR):
+    """Container for Time-Frequency data on epochs
 
-def write_tfrs(fname, tfr, overwrite=False):
-    """Write a TFR dataset to hdf5.
+    Can for example store induced power at sensor level or inter-trial
+    coherence.
```

Comment:
I think he's not talking about IO, but rather the idea that EpochsTFR cannot store induced power or ITC because these require computation across epochs, but this docstring makes it seem like it can (because it was copied from elsewhere)